tests/test_process_functions.py

```diff
@@ -420,3 +421,84 @@ def test_create_new_fileformat_table(syn):
             "newdb_mappingdf": update_return,
             "moved_ent": move_entity_return,
         }
+
+
+class TestCheckColAndValues:
+    @pytest.mark.parametrize(
+        "test_input",
+        [
+            pd.DataFrame({"some_col": ["Val1", "Val2", "val1"]}),
+            pd.DataFrame({"some_col": ["Val1", "Val2", "Val3"]}),
+            pd.DataFrame({"some_col": [None, "Val1", "Val2"]}),
+            pd.DataFrame({"some_col": [np.nan, "Val1", "Val2"]}),
```

Comment:
Nit:  This way you can remove the `import numpy as np` which is a larger import.
```suggestion
            pd.DataFrame({"some_col": [float('nan'), "Val1", "Val2"]}),
```